### DISCUSSION TITLE
bash script not available in the version 0.1.63

### DISCUSSION THREAD

**Discussion by Naranthiran** [ComplianceAsCode/content#10236] [Q&A] [answered]
Description of problem:
I am not able to find the bash scrip in the path /usr/share/scap-security-guide/bash/.

SCAP Security Guide Version:
0.1.63

Operating System Version:
RHEL 9.1

Steps to Reproduce:
1.install the below list of packages
openscap-1.3.6-4.el9.x86_64
openscap-scanner-1.3.6-4.el9.x86_64
scap-security-guide-0.1.63-5.el9.noarch

Any reason why the bash script is not available?

Regards
Naranthiran Duraisamy

**Comment by ggbecker** [comment, marked as answer]
The bash scripts were intentionally removed to reduce the footprint of the package since those scripts can be easily generated by running for example the following command:

`oscap xccdf generate fix --fix-type bash --profile stig /usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml > stig.sh`

You can simply change the profile to generate scripts for other profiles.

You can find out which profiles are available by running:

`oscap info /usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml`

**Reply by Naranthiran** [reply]
Hi ggbecker,

Thanks for your reply..

Regards
Naranthiran Duraisamy
